content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -304,7 +304,13 @@ mvn -Dset.changelist \
 
 == Troubleshooting
 
-Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets and that your pull request has a label indicating it was of interest to users (e.g., enhancement rather than chore).
+Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets.
+
+=== No release has been triggered after merging a pull request
+
+Ensure that the pull request has a label indicating it was of interest to users (e.g., `enhancement` rather than `chore`).
+
+See the complete list of label categories at https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml and the list of "interesting" categories at https://github.com/jenkins-infra/interesting-category-action/blob/main/action.yaml#L13.
 
 === The upload to the Maven repository fails with "401 Unauthorized"
```

Comment:
```suggestion
If you add an "interesting" label on a pull request already merged then merge another pull request (even with an uninteresting label), you will get a release since the workflow will see that there has been at least one interesting pull request since the last release.

Finally, if you manually trigger the CD action it will publish regardless of any labels. (May cause cause Release Drafter to rearrange release notes)

=== The upload to the Maven repository fails with "401 Unauthorized"
```